test/tests/transfers/multiapp_reporter_transfer/dist_vector/tests

```diff
@@ -0,0 +1,33 @@
+[Tests]
+    issues = '#26953'
+    design = 'MultiAppReporterTransfer.md'
+    [vec_rep]
+        requirement = "The system shall support the ability to transfer reporter data in a N to 1 and 1 to N fashion when"
+      [reporter_transfer]
+        type = 'JSONDiff'
+        input = 'main.i'
+        recover = false
+        jsondiff = 'main_out.json main_out_sub0_out.json  main_out_sub1_out.json main_out_sub2_out.json  main_out_sub3_out.json'
+        detail = "the number of procs per app is default,"
+      []
+      [reporter_transfer_min_procs_2]
+        type = 'JSONDiff'
+        cli_args = "MultiApps/sub/min_procs_per_app=2"
+        input = 'main.i'
+        jsondiff = 'main_out.json main_out_sub0_out.json  main_out_sub1_out.json main_out_sub2_out.json  main_out_sub3_out.json'
+        min_parallel = 2
+        detail = "the number of procs per app is less than the number of apps,"
+      []
+      [reporter_transfer_min_procs_5]
+        type = 'JSONDiff'
+        cli_args = "MultiApps/sub/min_procs_per_app=5"
+        input = 'main.i'
+        jsondiff = 'main_out.json main_out_sub0_out.json  main_out_sub1_out.json main_out_sub2_out.json  main_out_sub3_out.json'
+        min_parallel = 5
+        recover = false
```

Comment:
I need to turn off recover for all but 1 test because they try to use the same checkpoint and error out.

Reply:
Add the reason why "recover=false" as a comment in the tests file.

Reply:
Would this mean then I would have a set of gold files for every file_base?
` jsondiff = 'main_out.json main_out_sub0_out.json  main_out_sub1_out.json main_out_sub2_out.json  main_out_sub3_out.json'`.
I wanted to make sure that we are testing different parallel combinations on the same set of results that a serial run gave. I am having problems with the sub apps taking the same base name as the checkpoint files and don't know how to have both a different checkpoint name and keep the sub json files the same.